README.md

```diff
@@ -1,2 +1,105 @@
+
+The Docker image bundles a default `fluent.conf` as well as other import-able fluentd config files.
+
+The Kubernetes assets are a good example of overriding the default `fluent.conf` and importing configurations for things such as gathering pod logs from kubernetes or journald logs from systemd.
+
+**Plugins**
+
+This Docker image bundles the following (optional) fluentd plugins:
+- [fluent-plugin-papertrail](https://github.com/solarwinds/fluent-plugin-papertrail)
+- [fluent-plugin-loggly-syslog](https://github.com/solarwinds/fluent-plugin-loggly-syslog)
+- [fluent-plugin-systemd](https://github.com/reevoo/fluent-plugin-systemd)
+- [fluent-plugin-kubernetes_metadata_input](https://github.com/ViaQ/fluent-plugin-kubernetes_metadata_input)
+- [fluent-plugin-kubernetes_metadata_filter](https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter)
+
+The papertrail plugin allows us to treat Papertrail accounts as outputs.
+
+The loggly-syslog plugin allows us to treat Loggly accounts as outputs using the syslog protocol.
+
+The systemd plugin allows us to treat a host's journald logs as fluent input. The image is based on Debian, so that we can easily bundle the required systemd libraries.
+
+The kubernetes_metadata_input plugin lets us treat the Kubernetes Event API as fluent input.
+
+The kubernetes_metadata_filter plugin lets us recognize and bind Kubernetes specific metadata to logs from Kubernetes pods.
+
+## Advanced Usage
+
+### Kubernetes Annotations
+
+**Papertrail**
+
+Once the DaemonSet is running on your cluster, you can redirect logs to alternate Papertrail destinations by adding annotations to your Pods or Namespaces:
+
+```
+solarwinds.io/papertrail_host: 'logs0.papertrailapp.com'
+solarwinds.io/papertrail_port: '12345'
+```
+
+If both the Pod and Namespace have annotations for any running Pod, the Pod's annotation is used.
+
+**Loggly**
+
+Once the DaemonSet is running on your cluster, you can redirect logs to alternate Loggly destinations by adding annotations to your Pods or Namespaces:
+
+```
+solarwinds.io/loggly_token: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'
+```
+
+If both the Pod and Namespace have annotations for any running Pod, the Pod's annotation is used.
+
+### Kubernetes Audit Logs
+
+If you'd like to redirect Kubernetes API Server Audit logs to a seperate Papertrail or Loggly destination, add a second match statement to your `fluent.conf`:
+```
+<match kube-apiserver-audit>
+    type papertrail
+    num_threads 4
+
+    papertrail_host "#{ENV['FLUENT_PAPERTRAIL_AUDIT_HOST']}"
+    papertrail_port "#{ENV['FLUENT_PAPERTRAIL_AUDIT_PORT']}"
+</match>
+```
+
+This requires you to configure an [audit policy file](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/) on your cluster.
+
+## Development
+
+We have a [Makefile](Makefile) to wrap common functions and make life easier.
+
+### Release in [Quay.io](https://quay.io/repository/solarwinds/fluentd-kubernetes)
```

Comment:
While we've yet convene on the subject, we'll likely converge on Docker Hub (solarwinds org) as the consolidated, public repository for distribution of solarwinds images (publicly). How much effort is it to retool here?

Reply:
Not hard at all. As soon as I can get access to push images to Solarwinds org on DockerHub I can cut it over.